examples/webserver/cgi.c

```diff
@@ -80,7 +81,7 @@ static void net_stats(struct httpd_state *pstate, char *ptr)
   chunked_http_tx = pstate->ht_chunked;
 #endif
 
-  for (i = 0; i < sizeof(g_netstats) / sizeof(net_stats_t); i++)
+  for (i = 0; i < nitems(g_netstats); i++)
```

Comment:
This part will not work with `nitems` since here the structure members are iterated and `g_netstats` is not an array.
Let's revert changes in this file

Reply:
Done